draft-ietf-sframe-enc.md

```diff
@@ -1299,65 +1285,65 @@ In audio streams, there is typically a one-to-one relationship between frames
 and packets, so the overhead is the same whether one uses SFrame at a per-packet
 or per-frame level.
 
-The below table considers three scenarios, based on recommended configurations
+{{audio-overhead}} considers three scenarios that are based on recommended configurations
 of the Opus codec {{?RFC6716}}:
 
-* Narrow-band speech: 120ms packets, 8Kbps
-* Full-band speech: 20ms packets, 32Kbps
-* Full-band stereo music: 10ms packets, 128Kbps
+* Narrow-band (NB) speech: 120 ms packets (8 frames per second (fps)), 8 kbps
```

Comment:
```suggestion
* Narrow-band (NB) speech: 120 ms packets (8.3 frames per second (fps)), 8 kbps
```

? Why repeat this when it is in the table?

Reply:
Yeah, just to do the fps expansion outside the table.  I have reworked this table to remove the bullets entirely and put the "fps" expansion in the intro text.